ci-operator/config/openshift/ironic-image/openshift-ironic-image-release-4.11__prevalidation.yaml

```diff
@@ -0,0 +1,67 @@
+base_images:
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  ocp_builder_rhel-8-base-openshift-4.11:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-base-openshift-4.11
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base
+  inputs:
+    base:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-base-openshift-4.11
+  to: ironic
+promotion:
```

Comment:
Comparing the file to openshift-ironic-image-release-4.10__prevalidation.yaml , this bit is new, do you know what it means?

Reply:
since we're in 4.11 development cycle, this is to prevent the image created by this job to take place of the one created by master, so not promoting it
it will be removed once 4.11 becomes stable and 4.12 is master